From 1f4fecf46ba0cbdbbafb11b7295d231a11f661d4 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 4 Mar 2016 18:01:26 +0800
Subject: [PATCH] Revert "arm64: rockchip: rockchip_defconfig enable
 ARM_CPUIDLE and CPU_FREQ_DEFAULT_GOV_INTERACTIVE"

This reverts commit 754010a8f89cb3a5b04d3e749c2ef8396fac776b.

interactive driver will tirgger warning:
sysfs: cannot create duplicate filename '/devices/system/cpu/cpufreq'

Change-Id: I8074820953f2f76093df038f437890d65d054a28
---
 arch/arm64/configs/rockchip_defconfig | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c864ec7d88d3..7a68fd1596ab 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -59,13 +59,10 @@ CONFIG_PM_DEBUG=y
 CONFIG_PM_ADVANCED_DEBUG=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
 CONFIG_CPU_IDLE=y
-CONFIG_ARM_CPUIDLE=y
 CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
-CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
-CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_CPUFREQ_DT=y
 CONFIG_ARM_ROCKCHIP_CPUFREQ=y
@@ -454,6 +451,8 @@ CONFIG_USB_OTG_WAKELOCK=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_DEBUG_FILES=y
 CONFIG_USB_GADGET_VBUS_DRAW=500
+CONFIG_USB20_HOST=y
+CONFIG_USB20_OTG=y
 CONFIG_USB_CONFIGFS=y
 CONFIG_USB_CONFIGFS_F_FS=y
 CONFIG_USB_CONFIGFS_F_MTP=y
@@ -462,8 +461,6 @@ CONFIG_USB_CONFIGFS_F_ACC=y
 CONFIG_USB_CONFIGFS_F_AUDIO_SRC=y
 CONFIG_USB_CONFIGFS_UEVENT=y
 CONFIG_USB_CONFIGFS_F_MIDI=y
-CONFIG_USB20_HOST=y
-CONFIG_USB20_OTG=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
 # CONFIG_MMC_BLOCK_BOUNCE is not set
-- 
2.35.3

